java-client/src/test/java/org/scassandra/http/client/PrimingClientTest.java

```diff
@@ -921,4 +922,21 @@ public void testRetrievingOfMultiPreparedPrimesFailed() {
         //when
         underTest.retrieveQueryPrimes();
     }
+    
+    // bug fix https://github.com/scassandra/scassandra-server/issues/195
+    @Test
+    public void primeWithEmptyRowsDoesntThrowNullPointer() {
+        //given
+        stubFor(post(urlEqualTo(PRIME_PREPARED_PATH)).willReturn(aResponse().withStatus(200)));
+        PrimingRequest pr = PrimingRequest.preparedStatementBuilder()
+                .withQuery("select * from people")
+                .withThen(PrimingRequest.then().withVariableTypes(TEXT))
```

Comment:
Variable types are the ? in prepared statements, yours has none so can remove the withVariableTypes. This is a stubbed tests so doesn't really matter but the real server would reject this priming request.